HLTrigger/Configuration/python/customizeHLTTrackingForPhaseI2017.py

```diff
@@ -25,6 +25,15 @@ def modifyHLTPhaseIPixelGeom(process):
 
 # modify the HLT configuration to run the Phase I tracking in the particle flow sequence
 def customizeHLTForPFTrackingPhaseI2017(process):
+    # Bit of a hack to replace a module/sequence with another without
+    # expanding the sequences containing it, but works
+    #
+    # In principle setattr(process) can be used to replace a
+    # module/sequence too, but it expands the sequences making life
+    # more complex
+    modifier = cms.Modifier()
```

Comment:
@davidlange6 The discussion in these two issues #15425 and #15792 is related to automatic sequence expansion.